app/marko/src/client/preview/render.js

```diff
@@ -0,0 +1,99 @@
+/* global document */
```

Comment:
After the latest changes @Hypnosphi did, most of this code is redundant (moved to core) as well as "NoPreview.marko" and "ErrorDisplay.marko". take a look at the other apps to find the change.